code/game/turfs/simulated/walls.dm

```diff
@@ -198,7 +187,7 @@
 			return FALSE
 
 		to_chat(user, "<span class='notice'>You begin fixing dents on the wall...</span>")
-		if(W.use_tool(src, user, slicing_duration, volume=100))
+		if(W.use_tool(src, user, 0, volume=100))
```

Comment:
why did you remove slicing_duration

Reply:
It only had it because I forgot to remove when I added this
This is purely visual, having to waste welding fuel is already a fair price for it